images/linux/toolcache-1804.json

```diff
@@ -0,0 +1,14 @@
+{
+    "toolcache-python-ubuntu-1804-x64": [
+        "2.7", "3.5", "3.6", "3.7", "3.8"
+    ],
+    "toolcache-ruby-ubuntu-1804-x64": [
+        "2.4", "2.5", "2.6"
```

Comment:
Ruby 2.7 has been released recently
https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released/

Reply:
Ruby 2.7 support added. Additionally, Ruby 2.6.4 and 2.5.6 was also generated and published.